tests/lib/nm/wired_test.py

```diff
@@ -49,10 +49,21 @@ def test_create_setting_mtu(NM_mock):
 
 
 def test_create_setting_auto_negotiation_False(NM_mock):
-    setting = nm.wired.create_setting(
-        {'ethernet': {'auto-negotiation': False}}, None)
+    def mock_ethtool_result(device):
+        assert device == "nmstate_test"
```

Comment:
- This is already checked in line 66.
- Please use single quotes.
- I think you can use return_value and avoid the function + place the mock as a decorator: nicer structure.

Reply:
Handled all three items and squashed changes together in a single commit.